Vagrantfile

```diff
@@ -159,6 +159,7 @@ Vagrant.configure(2) do |config|
     elk.vm.hostname = 'elk.vagrant'
     elk.vm.network "private_network", ip: "10.0.0.104"
     elk.vm.network "forwarded_port", guest: 80, host: 8084
+    elk.vm.network "forwarded_port", guest: 5601, host: 5601
```

Comment:
This is a stale change I meant to make a few days ago. It allows a dev to hit kibana directly rather than going through the apache reverse proxy. We added a default deny vhost to the apache config here, so you need a hostname match to hit the correct vhost to proxy to kibana.